src/iai_bullet_sim/basic_simulator.py

```diff
@@ -97,22 +98,21 @@ def __leq__(self, other):
 
 class BasicSimulator(object):
     """Class wrapping the PyBullet interface in an object oriented manner."""
-    def __init__(self, tick_rate=50, gravity=[0,0,-9.81], use_egl=False):
+    def __init__(self, step_frequency=20, gravity=[0,0,-9.81], use_egl=False):
         """Constructs a simulator.
 
-        :param tick_rate: Ticks ideally performed per second.
-        :type  tick_rate: float
+        :param step_frequency: The frequency at which we will step forward. The internal simulation runs at 240 Hz, which needs to be a multiple of step_frequency 
```

Comment:
I'd prefer "step_frequency must be a divisor of 240, as this is the internal simulation frequency of bullet"